app/apps-v2.json

```diff
@@ -76,5 +76,65 @@
       "type": "addon",
       "url": "https://codingfree.com/addons/IMGUR/",
       "revision": "c47fec31d41cd526f7c10c94730ac8fa1c2489b2"
+    },
+    "https://elin-moco.github.io/fxos-addon-sound-switch/update.webapp": {
+      "manifestURL": "https://elin-moco.github.io/fxos-addon-sound-switch/update.webapp",
+      "name": "Sound Switch",
+      "description": "Toggle sounds on and off in utilities tray. Only media volume and ringtones/notifications volume would be affected.",
+      "icon": "https://elin-moco.github.io/fxos-addon-sound-switch/icons/sound.png",
+      "author": "Eddie Lin",
+      "type": "addon",
+      "url": "https://github.com/elin-moco/fxos-addon-sound-switch",
+      "revision": "3e136d786390fcbd874596693646c708c44568f5"
+    },
+    "https://elin-moco.github.io/fxos-addon-big-unlocker/update.webapp": {
+      "manifestURL": "https://elin-moco.github.io/fxos-addon-big-unlocker/update.webapp",
+      "name": "Bigger Unlocker",
+      "description": "Enlarges the unlocker on lockscreen.",
+      "icon": "https://elin-moco.github.io/fxos-addon-big-unlocker/icons/unlock.png",
+      "author": "Eddie Lin",
+      "type": "addon",
+      "url": "https://github.com/elin-moco/fxos-addon-big-unlocker",
+      "revision": "bffe63cce5d05c133b6531cae5416f5826b5c4eb"
+    },
+    "https://elin-moco.github.io/fxos-addon-draggable-home-btn/update.webapp": {
+      "manifestURL": "https://elin-moco.github.io/fxos-addon-draggable-home-btn/update.webapp",
+      "name": "Draggable Home Button",
+      "description": "Adds a draggable home button to the screen.",
+      "icon": "https://elin-moco.github.io/fxos-addon-draggable-home-btn/icons/home.png",
+      "author": "Eddie Lin",
+      "type": "addon",
+      "url": "https://github.com/elin-moco/fxos-addon-draggable-home-btn",
+      "revision": "0526c2e4f0b5407098dc43f0f0adfc43fe9d75ab"
+    },
+    "https://elin-moco.github.io/fxos-addon-yellow-mask/update.webapp": {
+      "manifestURL": "https://elin-moco.github.io/fxos-addon-update-mask/manifest.webapp",
```

Comment:
one of these uses `...update.manifest` and one uses `...manifest.webapp`. I'm still merging this PR, but I removed the yellow mask add-on.